src/python_testing/TC_RR_1_1.py

```diff
@@ -747,9 +759,13 @@ def build_acl(self):
         # Administer ACL entry
         admin_subjects = [0xFFFF_FFFD_0001_0001, 0x2000_0000_0000_0001, 0x2000_0000_0000_0002, 0x2000_0000_0000_0003]
 
+        admin_target_field_2 = Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_BC30)
+        if enable_access_to_group_cluster:
+            admin_target_field_2 = Clusters.AccessControl.Structs.Target(cluster=0x0000_0004)
+
         admin_targets = [
             Clusters.AccessControl.Structs.Target(endpoint=0),
-            Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_BC30),
+            admin_target_field_2,
             Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC01, deviceType=0xFFF1_BC31)
```

Comment:
If we wanna be precise regarding Master test plan the Targets shall be  
`Clusters.AccessControl.Structs.Target(cluster=0xFFF1_FC00, deviceType=0xFFF1_FC31)`